src/test/java/org/broadinstitute/consent/http/resources/DatasetResourceTest.java

```diff
@@ -201,74 +201,6 @@ public void testUpdateDatasetNoContent() {
-        result.setErrors(Collections.singletonList("Error!"));
-        when(api.overwrite(any(), any())).thenReturn(result);
-        when(api.create(any(), any())).thenReturn(result);
-
-        initResource();
-        Response response = resource.createDataSet(is, fileDataBodyPart, 1, false);
-        assertEquals(400, response.getStatus());
-    }
-
-    @Test
-    public void testCreateMissingHeaders() throws Exception {
-        File file = new File(ResourceHelpers.resourceFilePath(MISSING_HEADER));
-        MultiPart mp = createFormData(file);
-        FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("data", file, MediaType.valueOf("text/plain"));
-        InputStream is = IOUtils.toInputStream(mp.toString(), Charset.defaultCharset());
-        ParseResult result = new ParseResult();
-        result.setDatasets(Collections.emptyList());
-        result.setErrors(Collections.singletonList("Error!"));
-        when(api.overwrite(any(), any())).thenReturn(result);
-        when(api.create(any(), any())).thenReturn(result);
-
-        initResource();
-        Response response = resource.createDataSet(is, fileDataBodyPart, 1, false);
-        assertEquals(400, response.getStatus());
-    }
-
-    @Test
-    public void testCreateConsentIdAlreadyExists() throws Exception {
-        File file = new File(ResourceHelpers.resourceFilePath(WRONG_IDENTIFIERS));
-        MultiPart mp = createFormData(file);
-        FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("data", file, MediaType.valueOf("text/plain"));
-        InputStream is = IOUtils.toInputStream(mp.toString(), Charset.defaultCharset());
-        ParseResult result = new ParseResult();
-        result.setDatasets(Collections.emptyList());
-        result.setErrors(Collections.singletonList("Error!"));
-        when(api.overwrite(any(), any())).thenReturn(result);
-        when(api.create(any(), any())).thenReturn(result);
-
-        initResource();
-        Response response = resource.createDataSet(is, fileDataBodyPart, 1, false);
-        assertEquals(400, response.getStatus());
-    }
-
-    @Test
-    public void testCreateDataSet() throws IOException {
-        File file = new File(ResourceHelpers.resourceFilePath(CORRECT_FILE));
-        MultiPart mp = createFormData(file);
-        FileDataBodyPart fileDataBodyPart = new FileDataBodyPart("data", file, MediaType.valueOf("text/plain"));
-        InputStream is = IOUtils.toInputStream(mp.toString(), Charset.defaultCharset());
-        ParseResult result = new ParseResult();
-        result.setDatasets(Collections.emptyList());
-        result.setErrors(Collections.emptyList());
-        when(api.overwrite(any(), any())).thenReturn(result);
-        when(api.create(any(), any())).thenReturn(result);
-
-        initResource();
-        Response response = resource.createDataSet(is, fileDataBodyPart, 1, false);
-        assertEquals(200, response.getStatus());
-    }
-
```

Comment:
As a result of removing these tests, there is a good amount of unused imports and unused static strings. Please remove those as well.

Reply:
Unfortunately, even though `DataAccessRequestAPI dataAccessRequestAPI` isn't used in the test, it needs to be mocked for the resource class, so we need to keep that.